packages/template-retail-react-app/app/utils/marketing-cloud/marketing-cloud-email-link.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import crypto from 'crypto'
+import https from 'https'
+
```

Comment:
NIt: Can you drop some comments to explain the context of this file here? It seems this is an integration with another API for the pw reset. Do we have any related doc for that?

Reply:
We are going to include steps to configure Marketing Cloud email send as part of the feature docs

Reply:
https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/create-integration-enhanced.html
https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/transactional-messaging-api.html
These are some docs that Jinsu used to implement Marketing Cloud